src/pages/ProductListing/ProductListing.jsx

```diff
@@ -1,8 +1,43 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import axios from 'axios'
+import './ProductListing.css'
+import { ProductCard } from '../../components/ProductCard/ProductCard'
+import { useProduct } from '../../context/product-context'
+import { sortByPriceFunc , sortByRatingFunc , sortByCategoryFunc , priceRangeFilter } from '../../utilities/index'
+import { Nav } from '../../components/Nav/Nav'
+import { Filter } from '../../components/Filter/Filter'
```

Comment:
You can use an `index.js` file and add all your exports here and you can simply import all your component from a single file.